code/modules/mob/living/carbon/human/life.dm

```diff
@@ -1059,8 +1059,13 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 	return
 
 /mob/living/carbon/human/handle_changeling()
-	if(mind && mind.changeling)
-		mind.changeling.regenerate()
+	if(mind && hud_used)

```

Comment:
this is a bad place to put that check, changelings should continue working regardless of if they have a HUD or not